vhost_user_backend/src/lib.rs

```diff
@@ -113,6 +115,32 @@ pub trait VhostUserBackend: Send + Sync + 'static {
     /// A default implementation is provided as we cannot expect all backends
     /// to implement this function.
     fn set_slave_req_fd(&mut self, _vu_req: SlaveFsCacheReq) {}
+
+    /// Get epoll number.
+    /// While multiple queue are defined, it requires the backend to tell how many
+    /// vring handler thread it would like to run. The epoll number is not always
+    /// equal to the queue number, for net, it will be the count of queue pairs.
+    fn get_epoll_number(&self) -> usize;
+
+    /// Get vring subset.
+    /// While run multiple vring handler thread to service the multiple queues,
+    /// it requires the backend to how many queues each vring handler will handle.
+    fn get_vring_subset(
+        &self,
+        index: usize,
+        vrings: Vec<Arc<RwLock<Vring>>>,
+    ) -> Vec<Arc<RwLock<Vring>>>;
+
+    /// Get backend type
+    fn get_type(&self) -> u32 {
+        0
```

Comment:
In Rust style accessors should not really start with get_

Reply:
Drop get_type() it's a bad hack.

Reply:
Why do you think it is a bad hack?

Reply:
Anyway, this one is removed.